aeron-agent/src/main/java/io/aeron/agent/CommonEventDissector.java

```diff
@@ -144,4 +146,23 @@ else if (16 == addressLength)
 
         return encodedLength;
     }
+
+    static void appendFlagsAsChars(final short flags, final Appendable appendable)
```

Comment:
You can simply use `io.aeron.protocol.HeaderFlyweight#appendFlagsAsChars` method. Agent already pulls all of the decoders for the dissectors.